docs/source/whats_new.rst

```diff
@@ -27,6 +27,7 @@ Enhancements
 - Adding :func:`moabb.analysis.plotting.dataset_bubble_plot` plus the corresponding tutorial (:gh:`753` by `Pierre Guetschel`_)
 - Adding :func:`moabb.datasets.utils.plot_all_datasets` and update the tutorial (:gh:`758` by `Pierre Guetschel`_)
 - Improve the dataset model cards in each API page (:gh:`765` by `Pierre Guetschel`_)
+- Adding tutorial on using mne-features (:gh:`762` by `Alexander de Ranitz`_ and `Bruno Aristimunha`_)
```

Comment:
You can also add your other group members, depending on who actually contributed